sycl/test/basic_tests/esimd/vadd.cpp

```diff
@@ -9,19 +9,20 @@
 using namespace cl::sycl;
 
 class ESIMDSelector : public device_selector {
-  // Require GPU device unless HOST is requested in SYCL_DEVICE_TYPE env
+  // Require GPU device unless HOST is requested in SYCL_DEVICE_FILTER env
   virtual int operator()(const device &device) const {
-    if (const char *dev_type = getenv("SYCL_DEVICE_TYPE")) {
-      if (!strcmp(dev_type, "GPU"))
+    if (const char *dev_type = getenv("SYCL_DEVICE_FILTER")) {
+      if (!strcmp(dev_type, "gpu"))
```

Comment:
You change invalidated this code. `SYCL_DEVICE_FILTER` format ("backend:device_type:id") is different from `SYCL_DEVICE_TYPE` ("device_type").

`SYCL_DEVICE_FILTER` value can't be "gpu".